src/g3log/logmessage.hpp

```diff
@@ -87,7 +87,7 @@ namespace g3 {
       // Complete access to the raw data in case the helper functions above
       // are not enough.
       //
-      g3::system_time_point _timestamp;
+      g3::high_resolution_time_point _timestamp;
```

Comment:
It is not needed to change the name. It is hiding implementation details but I think that is intentional.